libflux/Dockerfile

```diff
@@ -33,7 +33,12 @@ ENV OPENSSL_LIB_DIR=/usr/local/ssl/lib \
     OPENSSL_STATIC=1
 
 # Install Clang
-RUN curl -SL https://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz \
+# Check https://releases.llvm.org/download.html#8.0.0 for the latest available binaries
+RUN curl https://releases.llvm.org/8.0.0/hans-gpg-key.asc | gpg --import - && \
+    curl -SL https://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz -O && \
+    curl -SL https://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz.sig -O && \
+    gpg --verify clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz.sig \
+                 clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz && \
     | tar -xJC . && \
```

Comment:
Hm... This line doesn't look right to me... Previously we were piping the tarball to `tar`...